### PR TITLE
TASK: Small css hacks for various styling issues

### LINKED ISSUE #1504: Small visual glitches in the tree
image

### PR DESCRIPTION
**Pull request by johannessteu**
Fixes #1504

### CODE REVIEW

**Comment by johannessteu** [comment]
Really hacky, but not that easy to solve at all in a "nicer" way

**Comment by johannessteu** [comment]
bildschirmfoto 2018-02-09 um 15 41 59

File: packages/neos-ui/src/Containers/LeftSideBar/index.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default class LeftSideBar extends PureComponent {
                     {LeftSideBarTop.map((Item, key) => <Item key={key} isExpanded={!isHiddenContentTree}/>)}
                 </div>
 
-                <hr/>
+                <hr style={{'border-top': 'none'}} />
```
Comment (dimaip) [inline packages/neos-ui/src/Containers/LeftSideBar/index.js]: Could also be `style={{borderTop: 'none'}}` I think.

**Comment by johannessteu** [comment]
Fixed @dimaip @mstruebing

File: packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/style.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .searchBar {
-    flex: 0 0 var(--spacing-GoldenUnit);
+    flex: 0 0 calc( var(--spacing-GoldenUnit) + 2px);
```
Comment (dimaip) [inline packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/style.css]: Still, an extra space here, the linter is compaining....

**Pull request merged**